src/main/java/com/codeofus/rent_a_park/controllers/PersonController.java

```diff
@@ -0,0 +1,50 @@
+package com.codeofus.rent_a_park.controllers;
+
+import com.codeofus.rent_a_park.dtos.ParkingMapper;
+import com.codeofus.rent_a_park.dtos.PersonDto;
+import com.codeofus.rent_a_park.models.Person;
+import com.codeofus.rent_a_park.services.PersonService;
+import lombok.AccessLevel;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@FieldDefaults(level = AccessLevel.PRIVATE, makeFinal = true)
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = "/api/v1/users")
+public class PersonController {
+
+    private PersonService personService;
```

Comment:
You use FieldDefaults with level = PRIVATE so noo need for **private** PersonService personService...